larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -34,6 +35,21 @@
 
 namespace tflite {
 
+constexpr int Padding_ONE = Padding_MAX + 1;
```

Comment:
what is `Padding_MAX`?

Reply:
The enum defines `Padding_MAX` as the maximum value of things in the enum, so that `MAX + 1` is always "unused".
We could change their enum to our own, but I think this will require changing a lot of code (I have not tested this particular change).